HoloPipelines/pipelines/pipelineAbdomenDicom2glb.py

```diff
@@ -0,0 +1,24 @@
+from components import compDicom2nifti
+from components import compHttpRequest
+from components import compNifti2numpy
+from components import compSeparateNumpy
+from components import compNumpyTransformation
+import sys
+
+
+def main(inputDicomPath, outputGlbFolderPath, segmentationModelUrl):
+    generatedNiftiPath = compDicom2nifti.main(inputDicomPath, "_temp.nii")
+    segmentedNiftiPath = compHttpRequest.sendFilePostRequest(
+        segmentationModelUrl, generatedNiftiPath, "_tempAbdomenSegmented.nii.gz"
+    )
+    generatedNumpyList = compNifti2numpy.main(segmentedNiftiPath)
+    resizedNumpyList = compNumpyTransformation.sizeLimit(generatedNumpyList, 300)
```

Comment:
Set 300 as the default value.  But leave it configurable as an argument in main()

Reply:
add 300 as default